### ISSUE TITLE
Installation Guide: Incomplete postgres config info prevents installation.

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#2289]
Two recent installations encountered issues completing installation due to incomplete postgres config info. It turns out not all the config info was copied over from v3.6. In particular, it only mentions the Unix sockets config needed by the installer but not the config needed for the app to communicate to the db:

At a minimum, if GlassFish is running on the same host, it will also need to allow password authentication on localhost. So you will need to modify the “host all all 127.0.0.1” line in your /var/lib/pgsq1/data/pg_hba.conf so that it looks like this:
host all all 127.0.0.1/32 password

There is more to add to this section not listed but refer to the postgres section of the v3.6 doc:
http://guides.dataverse.org/en/3.6.2/dataverse-installer-main.html#postgresql

Also, we never mention the password for the admin account here:
http://guides.dataverse.org/en/4.0/installation/installation-main.html

**Comment by pdurbin** [comment]
I can't take any credit for fixing this because I suspect @landreev is the author of http://guides.dataverse.org/en/4.2.3/installation/prerequisites.html#configure-database-access-for-the-dataverse-application but I further refined this section in my pull request at #2895 which can be previewed at http://guides.dataverse.org/en/2884-install-guide-reorg/installation/prerequisites.html#configure-database-access-for-the-dataverse-application

In short I believe this has already been fixed in 4.2.3 or earlier. Passing to QA.

**Comment by kcondon** [comment]
Yes, this was previously fixed. Closing.
